bazel/external/wamr_llvm.BUILD

```diff
@@ -10,30 +10,27 @@ filegroup(
 )
 
 cmake(
-    name = "llvm_13_0_1_lib",
+    name = "llvm_lib",
     cache_entries = {
         # Disable both: BUILD and INCLUDE, since some of the INCLUDE
         # targets build code instead of only generating build files.
+        # Dsiable unncessary components
         "LLVM_BUILD_BENCHMARKS": "off",
         "LLVM_INCLUDE_BENCHMARKS": "off",
         "LLVM_BUILD_DOCS": "off",
         "LLVM_INCLUDE_DOCS": "off",
         "LLVM_BUILD_EXAMPLES": "off",
         "LLVM_INCLUDE_EXAMPLES": "off",
-        "LLVM_BUILD_RUNTIME": "off",
-        "LLVM_BUILD_RUNTIMES": "off",
-        "LLVM_INCLUDE_RUNTIMES": "off",
         "LLVM_BUILD_TESTS": "off",
         "LLVM_INCLUDE_TESTS": "off",
         "LLVM_BUILD_TOOLS": "off",
         "LLVM_INCLUDE_TOOLS": "off",
-        "LLVM_BUILD_UTILS": "off",
-        "LLVM_INCLUDE_UTILS": "off",
```

Comment:
Any reason why those are removed?

Reply:
LLVM is keep improving its CMAKE build options. All removed are unused.